.travis.yml

```diff
@@ -1,4 +1,6 @@
-sudo: required
+dist: xenial
```

Comment:
Ok, makes sense, based on: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

Reply:
that doesn't really change anything as we would have been defaulted to using Xenial anyways, but the Tavis build validation feature I enabled pointed that out as an issue so I only added that to remove that warning.